.github/workflows/build.yml

```diff
@@ -176,11 +179,6 @@ jobs:
           registry: ${{ secrets.GCP_DOCKER_REGISTRY }}
           username: "oauth2accesstoken"
           password: "${{ steps.auth.outputs.access_token }}"
-      #
```

Comment:
chances are we'll never build for anything but linux, but just in case i moved the comments to an issue here:
documented here https://github.com/ParabolInc/parabol/issues/8929